rllib/env/multi_agent_env_runner.py

```diff
@@ -135,6 +137,93 @@ def __init__(self, config: AlgorithmConfig, **kwargs):
         # and receiving the next `sample()` request from the user.
         self._time_after_sampling = None
 
+    # @override(EnvRunner)
```

Comment:
Goes in the next commit. Not needed anymore. We do it now similar to the `SingleAgentEnvRunner`.